src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -515,6 +518,7 @@ public function setBodyFromClosure(Closure $newBody) : void
      */
     public function setBodyFromString(string $newBody) : void
     {
+        /** @var Node[] $this->node->stmts */
```

Comment:
I haven't checked, but does PhpStorm (or other IDEs) actually support this syntax? o_O also, can this not be inferred anyway? And finally, why are we documenting the type when it's not actually used anywhere later?

Reply:
No, this syntax is not supported

Reply:
I'll remove

> also, can this not be inferred anyway

The method can return null here